packages/core/vue/src/useWallet.ts

```diff
@@ -38,13 +37,13 @@ export interface WalletStore {
         options?: SendTransactionOptions
     ): Promise<TransactionSignature>;
 
-    signTransaction: Ref<SignerWalletAdapter['signTransaction'] | undefined>;
-    signAllTransactions: Ref<SignerWalletAdapter['signAllTransactions'] | undefined>;
-    signMessage: Ref<MessageSignerWalletAdapter['signMessage'] | undefined>;
+    signTransaction: Ref<SignerWalletAdapterProps['signTransaction'] | undefined>;
+    signAllTransactions: Ref<SignerWalletAdapterProps['signAllTransactions'] | undefined>;
+    signMessage: Ref<MessageSignerWalletAdapterProps['signMessage'] | undefined>;
```

Comment:
Can we keep these lines unchanged? The `Props` should probably have been an internal API.

Reply:
These have been changed back